### DISCUSSION TITLE
etsy all active listing showing same listings with offset

### DISCUSSION THREAD

**Discussion by Locgfxsaurabh** [etsy/open-api#1042] [Q&A]
etsy all active listing showing same listing in a new page with offset please help??

**Comment by emergingdzns** [comment]
@Locgfxsaurabh Can you post the url with input parameters and payload? We have no information to go on here. Could be a typo somewhere or something.

**Reply by Locgfxsaurabh** [reply]
https://openapi.etsy.com/v3/application/listings/active?limit=25&offset=4
this is the url its generated same listing except the top ones

**Reply by Locgfxsaurabh** [reply]
its returning same products after change offset

**Reply by emergingdzns** [reply]
Ok so if I'm understanding you correctly, your initial call is to `https://openapi.etsy.com/v3/application/listings/active?limit=25&offset=0`. Then if you call `https://openapi.etsy.com/v3/application/listings/active?limit=25&offset=4` you get all the same listings except for the top 4 correct?

If so that makes perfect sense. In your first call you're telling it you want 25 records. Then in the next call you're saying you want 25 records, but skip just the first 4. So the listings shift and you'll get 4 new listings at the end but the other 21 will be the same.

The offset value isn't a "page" value but the count you want to offset by. So page 2 would be `limit=25&offset=25`, page 3 would be `limit=25&offset=50` and page 4 would be `limit=25&offset=75`.

**Reply by Locgfxsaurabh** [reply]
can you help me with the pagination i want per page has 25 posts so what would be my url of api call

**Reply by Locgfxsaurabh** [reply]
Thanks alot its works for me..

**Reply by emergingdzns** [reply]
Great. Please mark the question as answered so that others know they can find answers here.

**Comment by Locgfxsaurabh** [comment]
Yess please help
